ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/139__trusted_ca_bundles.robot

```diff
@@ -25,6 +25,8 @@ Validate Trusted CA Bundles State Managed
     ...    each non-reserved namespace.
     [Tags]    Operator    Smoke    ODS-2638    TrustedCABundle-Managed
 
+    ${saved_custom_ca_bundle}=    Get Custom CA Bundle Value In DSCI     ${DSCI_NAME}    ${OPERATOR_NS}
```

Comment:
Calling this function in a suite setup would be even better, if possible.

Reply:
Do you have an example of how to save a value in a global, and then access the value in a test case (was not able to determine a working example)?

Reply:
Hm, I would expect that creating a `*** Variables ***` section and assigning a variable defined there a value in the setup function would work, see e.g. here https://docs.robotframework.org/docs/variables.

There is another possibility to use `Set Global Variable`, but I don't like that approach.

I can live with the variant as is right now, but would make sense to have it in some common setup phase instead in every test :slightly_smiling_face:

Reply:
There are additional test cases where saving/restoring state is also needed, and a better approach is needed. Similarly for Management state, and in particular when testing "Managed" and the start configuration is Removed or Unmanaged.

Reply:
yeah, can't disagree

Reply:
You can use `Set Suite Variable <var_name> <var_value>`

Reply:
Followup #1401